src/analysisd/makelists.c

```diff
@@ -173,7 +174,7 @@ int main(int argc, char **argv)
         Config.lists = NULL;
     }
 
-    Lists_OP_MakeAll(force);
-
+    Lists_OP_MakeAll(force, show_message);
+    printf(" * The CBD list will be compiled automatically at restart of Wazuh\n");
```

Comment:
Please change log message to
printf(" * The CBD list will be compiled automatically once Wazuh is restarted.\n");